libraries/c_sdk/standard/https/src/iot_https_client.c

```diff
@@ -2694,7 +2659,7 @@ IotHttpsReturnCode_t IotHttpsClient_Disconnect( IotHttpsConnectionHandle_t connH
      * on this function that it can proceed with the disconnecting activities. */
     if( connHandle != NULL )
```

Comment:
How could this suddenly be NULL?

Reply:
If the user passes in a NULL connHandle to the routine. This is within the cleanup goto statements.